### PR TITLE
Added additional information to reward points pages

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
## Purpose of this pull request

Added additional information to reward points pages:
- using reward points during the checkout
- notice, that points in cart price rules applies only to registered users.

## Affected documentation pages

https://docs.magento.com/user-guide/customers/account-dashboard-reward-points.html
https://docs.magento.com/user-guide/marketing/reward-points-price-rules.html

whatsnew
Updated the [Reward Points](https://docs.magento.com/user-guide/customers/account-dashboard-reward-points.html) dashboard topic to include requirements and workflow for customers to use reward points during checkout.

### CODE REVIEW

File: src/customers/account-dashboard-reward-points.md [hunk 1]
```diff
@@ -37,6 +37,27 @@ The rewards points balance can be updated from the Admin.
...
+
+1. Click <span class="btn">Use reward points</span>.
+
+![]({% link customers/assets/reward-points-on-checkout.png %}){: .zoom}
+_Reward Points on checkout_
+
+The available points discount was applied to subtotal.
+
+{:.bs-callout-info}
+If the available balance is greater than the order grand total, no additional payment methods is required.
```
Comment (leslietilling) [inline src/customers/account-dashboard-reward-points.md:59]:
```suggestion
If the available balance is greater than the order grand total, an additional payment method is not required.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Some edits for style and clarity.

**Review by leslietilling — APPROVED** [review]

**Comment by m2-github-services** [comment]
@leslietilling an error occurred during the Pull Request import.

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@leslietilling an error occurred during the Pull Request import.

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@leslietilling an error occurred during the Pull Request import.

**Comment by sidolov** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@sidolov an error occurred during the Pull Request import.

**Comment by DmitryTsymbal** [comment]
Closing this one, and opening https://github.com/magento/merchdocs/pull/1690 to avoid an errors during the import

**Pull request closed without merging**